docs/src/main/paradox/release-notes/3.0.x/avroparquet.md

```diff
@@ -1,5 +1,11 @@
 # AvroParquet
 
+## 3.0.0
+
+- Enable fatalwarnings for most Alpakka and internal projects [#2628](https://github.com/akka/alpakka/issues/2628) by [@seglo](https://github.com/seglo)
+
+[*closed in 3.0.0-M1*](https://github.com/akka/alpakka/issues?q=is%3Aclosed+milestone%3A3.0.0+label%3Ap%3Aavroparquet)
```

Comment:
```suggestion
[*closed in 3.0.0*](https://github.com/akka/alpakka/issues?q=is%3Aclosed+milestone%3A3.0.0+label%3Ap%3Aavroparquet)
```

Seems to be consistent like this for all connectors.

Reply:
:+1: